src/App.tsx

```diff
@@ -1,25 +1,39 @@
-import {NavigationBar} from "./Components/index"; 
+import { NavigationBar } from "./Components/index";
 import QuestionPage from "./Pages/QuestionsPage/QuestionPage";
-// import LandingPage from "./Pages/LandingPage/LandingPage";
+import LandingPage from "./Pages/LandingPage/LandingPage";
+import CategoryPage, {
+  CategoryContainer,
+} from "./Pages/CategoryPage/CategoryPage";
 import AuthenticationPage from "./Pages/AuthenticationPage/AuthenticationPage";
-import {db,auth} from "./fireBase";
-import {Routes,Route} from "react-router-dom";
+import { db, auth } from "./fireBase";
+import { Routes, Route } from "react-router-dom";
 import LogInForm from "Components/Authentication/LogInForm";
 import SignUpForm from "Components/Authentication/SignUpForm";
+import ResultsPage from "Pages/ResultsPage/ResultsPage";
```

Comment:
Suggestion: create an index file in the pages folder and export all the pages from the index. This would increase readability.

Reply:
I would do this later on as of now I m in a bit hurry for my submissions